### PR TITLE
DOC: Documentation updates for devtron installation, ingess, liveness and readiness probe

### PR DESCRIPTION
**Pull request by Shubham9t9**
# Description

Updates in documentation:
1. Added how we can see logs of devtron installer while installing devtron.
2. Added how we can apply liveness & readiness probes by executing commands.
3.  Added `pathType` in ingress.

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] This change requires a documentation update

# Checklist:

* [ ] The title of the PR states what changed and the related issues number (used for the release note).
* [X] Does this PR require documentation updates?
* [X] I've updated documentation as required by this PR.
* [ ] I have performed a self-review of my own code
* [ ] I have commented my code, particularly in hard-to-understand areas
* [ ] I have tested it for all user roles

### CODE REVIEW

File: docs/setup/install/install-devtron-helm-2.md [hunk 1]
```diff
@@ -70,6 +70,11 @@ kubectl -n devtroncd get installers installer-devtron -o jsonpath='{.status.sync
 ```
 
 The install commands initiates Devtron-operator which spins up all the Devtron micro-services one by one in about 20 mins. You can use the above command to check the status of the installation if the installation is still in progress, it will print `Downloaded`. When the installation is complete, it prints `Applied`.
+During devtron installation process, you can check logs of installer. To check logs of installer, run the following command :
```
Comment (arushi09-hub) [inline docs/setup/install/install-devtron-helm-2.md:73]: Replace this line with " Meanwhile, you can check logs of the installer, by executing the following command

File: docs/setup/install/install-devtron-helm-3.md [hunk 2]
```diff
@@ -63,6 +63,11 @@ kubectl -n devtroncd get installers installer-devtron -o jsonpath='{.status.sync
 ```
 
 The install commands initiates Devtron-operator which spins up all the Devtron micro-services one by one in about 20 mins. You can use the above command to check the status of the installation if the installation is still in progress, it will print `Downloaded`. When the installation is complete, it prints `Applied`.
+During devtron installation process, you can check logs of installer. To check logs of installer, run the following command :
```
Comment (arushi09-hub) [inline docs/setup/install/install-devtron-helm-3.md:66]: Replace the content, as mentioned above.

File: docs/setup/install/install-devtron-using-kubectl.md [hunk 3]
```diff
@@ -35,7 +35,11 @@ kubectl -n devtroncd get installers installer-devtron -o jsonpath='{.status.sync
 ```
 
 The install commands initiates Devtron-operator which spins up all the Devtron micro-services one by one in about 20 mins. You can use the above command to check the status of the installation if the installation is still in progress, it will print `Downloaded`. When the installation is complete, it prints `Applied`.
+During devtron installation process, you can check logs of installer. To check logs of installer, run the following command :
```
Comment (arushi09-hub) [inline docs/setup/install/install-devtron-using-kubectl.md:38]: Replace the content as mentioned above.

File: docs/user-guide/creating-application/deployment-template/rollout-deployment.md [hunk 4]
```diff
@@ -71,6 +71,9 @@ LivenessProbe:
   successThreshold: 1
   timeoutSeconds: 5
   failureThreshold: 3
+  command:
```
Comment (arushi09-hub) [inline docs/user-guide/creating-application/deployment-template/rollout-deployment.md:74]: Why is this added? Please explain in the description of the PR.

File: docs/user-guide/creating-application/deployment-template/rollout-deployment.md [hunk 5]
```diff
@@ -141,6 +148,7 @@ ReadinessProbe:
 | `successThreshold` | It defines the number of successes required before a given container is said to fulfill the readiness probe. |
 | `timeoutSeconds` | It defines the time for checking timeout. |
 | `failureThreshold` | It defines the maximum number of failures that are acceptable before a given container is not considered as ready. |
+| `command` | The mentioned command is executed to perform the livenessProbe. |
```
Comment (arushi09-hub) [inline docs/user-guide/creating-application/deployment-template/rollout-deployment.md:151]: Please review this `command` once again from [here](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/)

File: docs/user-guide/creating-application/deployment-template/rollout-deployment.md [hunk 6]
```diff
@@ -249,19 +259,22 @@ ingressInternal:
...
 ```
 
 | Key | Description |
 | :--- | :--- |
 | `enabled` | Enable or disable ingress |
 | `annotations` | To configure some options depending on the Ingress controller |
-| `path` | Path name |
 | `host` | Host name |
+| `path` | Path name |
+| `pathType` | Path type. Supported path types are `ImplementationSpecific`, `Exact` and `Prefix`.|
```
Comment (arushi09-hub) [inline docs/user-guide/creating-application/deployment-template/rollout-deployment.md:277]: Only write, "Supported path types are `ImplementationSpecific`, Exact` and `Prefix`

**Comment by arushi09-hub** [comment]
Please write proper description & Title of the PR

File: docs/user-guide/creating-application/deployment-template/rollout-deployment.md (hunk 4 above)
Comment (Shubham9t9) [reply docs/user-guide/creating-application/deployment-template/rollout-deployment.md:74]: No need to change

File: docs/setup/install/install-devtron-using-kubectl.md (hunk 3 above)
Comment (Shubham9t9) [reply docs/setup/install/install-devtron-using-kubectl.md]: done

File: docs/user-guide/creating-application/deployment-template/rollout-deployment.md (hunk 5 above)
Comment (Shubham9t9) [reply docs/user-guide/creating-application/deployment-template/rollout-deployment.md:151]: No need to change

File: docs/user-guide/creating-application/deployment-template/rollout-deployment.md (hunk 6 above)
Comment (Shubham9t9) [reply docs/user-guide/creating-application/deployment-template/rollout-deployment.md]: done

File: docs/setup/install/install-devtron-helm-3.md (hunk 2 above)
Comment (Shubham9t9) [reply docs/setup/install/install-devtron-helm-3.md]: done

File: docs/setup/install/install-devtron-helm-2.md (hunk 1 above)
Comment (Shubham9t9) [reply docs/setup/install/install-devtron-helm-2.md]: done

File: docs/user-guide/creating-application/deployment-template/rollout-deployment.md (hunk 4 above)
Comment (prakarsh-dt) [reply docs/user-guide/creating-application/deployment-template/rollout-deployment.md:74]: Use
```
  command:
    - python
    - /etc/app/healthcheck.py
   ```

File: docs/user-guide/creating-application/deployment-template/rollout-deployment.md [hunk 7]
```diff
@@ -126,6 +130,9 @@ ReadinessProbe:
   successThreshold: 1
   timeoutSeconds: 5
   failureThreshold: 3
+  command:
```
[the remark this answers is not in the document]
Comment (Shubham9t9) [reply docs/user-guide/creating-application/deployment-template/rollout-deployment.md:133]: done

File: docs/user-guide/creating-application/deployment-template/rollout-deployment.md [hunk 8]
```diff
@@ -86,6 +89,7 @@ LivenessProbe:
 | `successThreshold` | It defines the number of successes required before a given container is said to fulfil the liveness probe. |
 | `timeoutSeconds` | It defines the time for checking timeout. |
 | `failureThreshold` | It defines the maximum number of failures that are acceptable before a given container is not considered as live. |
+| `command` | The mentioned command is executed to perform the livenessProbe. |
```
[the remark this answers is not in the document]
Comment (Shubham9t9) [reply docs/user-guide/creating-application/deployment-template/rollout-deployment.md]: done

File: docs/user-guide/creating-application/deployment-template/rollout-deployment.md [hunk 9]
```diff
@@ -141,6 +148,7 @@ ReadinessProbe:
 | `successThreshold` | It defines the number of successes required before a given container is said to fulfill the readiness probe. |
 | `timeoutSeconds` | It defines the time for checking timeout. |
 | `failureThreshold` | It defines the maximum number of failures that are acceptable before a given container is not considered as ready. |
+| `command` | The mentioned command is executed to perform the readinessProbe. |
```
[the remark this answers is not in the document]
Comment (Shubham9t9) [reply docs/user-guide/creating-application/deployment-template/rollout-deployment.md]: done

File: docs/user-guide/creating-application/deployment-template/rollout-deployment.md [hunk 10]
```diff
@@ -208,10 +216,12 @@ ingress:
       - host: example1.com
         paths:
             - /example
+        pathType:
```
Comment (prakarsh-dt) [inline docs/user-guide/creating-application/deployment-template/rollout-deployment.md:219]: Put this before paths field to remove confusion and what about the value of pathType:?
Example:

```
  hosts:
      - host: example1.com
        pathType: "ImplementationSpecific"
        paths:
            - /example
```

Comment (prakarsh-dt) [reply docs/user-guide/creating-application/deployment-template/rollout-deployment.md:219]: Repeat all all places

Comment (Shubham9t9) [reply docs/user-guide/creating-application/deployment-template/rollout-deployment.md]: done

**Review by prakarsh-dt — APPROVED** [review]
